web/themes/custom/bglobal/templates/paragraphs/paragraph--hero.html.twig

```diff
@@ -0,0 +1,75 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a paragraph.
+ *
+ * Available variables:
+ * - paragraph: Full paragraph entity.
+ *   Only method names starting with "get", "has", or "is" and a few common
+ *   methods such as "id", "label", and "bundle" are available. For example:
+ *   - paragraph.getCreatedTime() will return the paragraph creation timestamp.
+ *   - paragraph.id(): The paragraph ID.
+ *   - paragraph.bundle(): The type of the paragraph, for example, "image" or "text".
+ *   - paragraph.getOwnerId(): The user ID of the paragraph author.
+ *   See Drupal\paragraphs\Entity\Paragraph for a full list of public properties
+ *   and methods for the paragraph object.
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *   
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{% set classes = [
+  'paragraph',
+  'paragraph--type-' ~ paragraph.bundle|clean_class,
+  view_mode ? 'paragraph--display-mode-' ~ view_mode|clean_class,
+] %}
+
+{% set attributes = attributes.addClass(classes) %}
+
+{% set image_url = file_url(content.field_image.0['#media'].field_media_image.entity.uri.value) %}
+
+{% set heroTitle %}
+  {{ content.field_title }}
+{% endset %}
+
+{% set heroSubtitle %}
+  {{ content.field_subtitle }}
+{% endset %}
+
+{% block content %}
+  {% include '@molecules/hero/hero.twig' with {
+    'hero': {
+      'img': {
+        'src': image_url,
+        'alt': content.field_image.0['#media'].field_media_image.alt
+      }, 
+      'title': heroTitle | striptags,
```

Comment:
Since the title doesn't use HTML tags, another approach here would be to extract the string value of this field. (probably something like `content.field_title.0.value`)